### ISSUE TITLE
Incomplete instructions to compile in GNU/Linux?

### ISSUE THREAD

**Issue by cpcbegin** [laullon/b2t80s#4]
I try to compile the emulator in Ubuntu Mate 18.04 64 bits with these commands:

```
sudo apt install go-bindata  golang-go git
git clone https://github.com/laullon/b2t80s
go get -u github.com/go-bindata/go-bindata/...
go-bindata -pkg data -o data/data.go data/...
```

And I try to run a ZX Spectrum game with the command:
`go run main.go --mode 48k -tap "/home/miuser/zxspectrum/homebrew/altair_zx_es.tap"`

But I get this error:
`error cannot find package "fyne.io/fyne"`

**Comment by laullon** [comment]
Thanks for reporting, will take a look.

**Comment by laullon** [comment]
you can download the binary emulator from here: https://github.com/laullon/b2t80s/releases

**Comment by cpcbegin** [comment]
Thanks! ;-)

**Comment by laullon** [comment]
Which go version are you using ?
It should be go v1.13

**Comment by cpcbegin** [comment]
I'm using the go version included in Ubuntu 18.04 repositories:
go version go1.10.4 linux/amd64
